src/Microsoft.OData.Core/UriParser/Resolver/ODataUriResolver.cs

```diff
@@ -504,7 +503,22 @@ internal static ODataUriResolver GetUriResolver(IServiceProvider container)
             return container.GetRequiredService<ODataUriResolver>();
         }
 
-        private static IList<T> FindAcrossModels<T>(IEdmModel model, String qualifiedName, bool caseInsensitive) where T : IEdmSchemaElement
+        private static IReadOnlyList<T> FindElementsCaseInsensitive<T>(

```

Comment:
```suggestion
        private static IReadOnlyList<T> FindElements<T>(
```
Or if you must be explicit about it being a case-insensitive search, `FindElementsIgnoreCase` might be a better fit

Reply:
I wasn't sure what call it cause there are other `Find*` methods: `FindAcrossModels` and `FindSchemaElements`. Maybe I should call the new method `FindSchemaElements` and rename the existing `FindSchemaElements` to `FindInModel` or `FindSchemaElementsInModel`.

Reply:
Don't forget the method is private...

Reply:
I renamed the `FindElementsCaseInsensitive` to `FindSchemaElements` and `FindSchemaElements` to `FindSchemaElementsInModel`